src/SignalR/common/Http.Connections/src/Internal/HttpConnectionDispatcher.Log.cs

```diff
@@ -44,7 +44,7 @@ internal static class Log
                 LoggerMessage.Define<HttpTransportType>(LogLevel.Trace, new EventId(11, "ReceivedDeleteRequestForUnsupportedTransport"), "Received DELETE request for unsupported transport: {TransportType}.");
 
             private static readonly Action<ILogger, Exception> _terminatingConnection =
-                LoggerMessage.Define(LogLevel.Trace, new EventId(12, "TerminatingConection"), "Terminating Long Polling connection due to a DELETE request.");
+                LoggerMessage.Define(LogLevel.Trace, new EventId(12, "TerminatingConnection"), "Terminating Long Polling connection due to a DELETE request.");
```

Comment:
Log IDs can't be changed, they're effectively public.

Reply:
I have just reverted it.